week8/trivia/index.js

```diff
@@ -0,0 +1,24 @@
+let correct = document.querySelector('.correct');
+correct.addEventListener('click', function() {
+    correct.style.backgroundColor = 'green';
+    document.querySelector('#feedback1').innerHTML = 'Correct!';
+});
+
+let incorrects = document.querySelectorAll('.incorrect');
+for (let i = 0; i < incorrects.length; i++) {
+    incorrects[i].addEventListener('click', function() {
+        incorrects[i].style.backgroundColor = 'red';
+        document.querySelector('#feedback1').innerHTML = 'incorrect';
+    });
+}
+
+document.querySelector('#check').addEventListener('click', function() {
+    let input = document.querySelector('input');
+    if (input.value === 'Berlin' || 'berlin') {
```

Comment:
You can call "toLowerCase" on the input value since it should be a string. Then just compare it with the lower case version of whatever you trying to find.